src/registrar/tests/test_admin.py

```diff
@@ -886,6 +888,86 @@ def test_get_filters(self):
         self.assertContains(response, retrieved_html, count=1)
 
 
+class UserDomainRoleAdminTest(TestCase):
+    def setUp(self):
+        """Setup environment for a mock admin user"""
+        self.site = AdminSite()
+        self.factory = RequestFactory()
+        self.admin = ListHeaderAdmin(model=UserDomainRoleAdmin, admin_site=None)
+        self.client = Client(HTTP_HOST="localhost:8080")
+        self.superuser = create_superuser()
+
+    def tearDown(self):
+        """Delete all Users, Domains, and UserDomainRoles"""
+        User.objects.all().delete()
+        Domain.objects.all().delete()
+        UserDomainRole.objects.all().delete()
+
+    def test_email_not_in_search(self):
+        """Tests the search bar in Django Admin for UserDomainRoleAdmin.
+        Should return no results for an invalid email."""
+        # Have to get creative to get past linter
+        p = "adminpass"
+        self.client.login(username="superuser", password=p)
+
+        fake_user = User.objects.create(
+            username="dummyuser", first_name="Stewart", last_name="Jones", email="AntarcticPolarBears@cold.com"
```

Comment:
**comment (non-blocking, pedantic)**: It minimizes the pun potential, but `cold.com` is potentially a real site and email addresses from their domain showing up in our code base seems strange. `example.com` is reserved for this purpose and will never be a real domain, so I think we should stick to using it as much as we can.

Reply:
Thanks for pointing that out, good points. Probably best to get that changed.

Turns out cold.com is a real site, rather cryptic if you ask me:
http://cold.com/